src/couple_terms.jl

```diff
@@ -1,8 +1,35 @@
+"""
+    couple_terms(t1, t2)
+
+Generate all possible coupling terms between `t1` and `t2`.  It is
+assumed that t1 and t2 originate from non-equivalent electrons, since
+the vector model does not predict correct term couplings for
```

Comment:
The vector model is the one we use to describe angular momentum coupling, e.g. j = l + s via vector addition. This is only approximately true, i.e. there is no perfectly LS-coupled system, etc. Hence "model".

Re equivalent electrons: I'm not sure if electrons of different ell or spin could be equivalent, since you could devise an experiment which is sensitive to ell (or spin). All electrons are _identical particles_, but I think equivalent means something slightly different. I have to think/read about it.

Reply:
Hmm, I indeed assumed that "equivalent" is synonymous with "identical", but it isn't.

Re "vector model": I would avoid the term here though. Could maybe go for something like:
```
Generate all possible coupling terms between `t1` and `t2` of _different_ subshells (i.e. of non-equivalent electrons).

!!! note
    Due to the Pauli principle, this _can not_ be used for coupling _within_ a subshell (cf. Cowan p. 108–109).
```